### PR TITLE
ability to browse yesterday's tasks and notes

### PR DESCRIPTION
**Pull request by karuvally**
#### Short description of what this resolves:
Inability of the user to quickly browse yesterday's tasks and notes

#### Changes proposed in this pull request:

- "yoda diary notes yesterday" gives yesterday's notes
- "yoda diary tasks yesterday" gives yesterday's tasks

### CODE REVIEW

File: modules/diary.py [hunk 1]
```diff
@@ -35,6 +35,14 @@ def now_date():
     return str(time.strftime("%d-%m-%Y"))
```
Comment (manparvesh) [inline modules/diary.py]: please remove all files other than this one, since they are not related to the fix.

**Comment by manparvesh** [comment]
@karuvally the fix looks fine to me, however, you have added some other files to the PR. Please remove them.

File: modules/diary.py (hunk 1 above)
Comment (karuvally) [reply modules/diary.py]: okay, will do that

Comment (manparvesh) [reply modules/diary.py]: Okay, I that case, I will close this PR

**Pull request closed without merging**